connections/simopen.md

```diff
@@ -0,0 +1,89 @@
+# Simultaneous Open for bootstrapping connections in multistream-select
```

Comment:
```suggestion
# Connection bootstrapping: Handling simultaneous open in multistream-select.
```

As per @raulk's comment in https://github.com/libp2p/specs/pull/196#discussion_r308391270.